packages/flutter/lib/src/material/text_selection.dart

```diff
@@ -32,8 +34,9 @@ class _TextSelectionToolbar extends StatelessWidget {
     }
     items.add(new FlatButton(
       child: new Text('PASTE'),
-      onPressed: delegate.pasteBuffer != null ? _handlePaste : null)
-    );
+      // TODO(mpcomplete): should this be grayed-out if there is nothing to paste?
```

Comment:
(Short answer: It's because when I'm reading the code, if I see a question, I usually don't have the context to answer the question. Phrasing it as a statement pushes one to support one's statement, which provides context for the reader.)